### ISSUE TITLE
[bug] Window content extending over the boundary (macOS)

### ISSUE THREAD

**Issue by vtu-dog** [tauri-apps/tauri#3794]
### Describe the bug

Hello,
I tried to create a wrapper for OvenPlayer and found some strange bugs related to windows. [Here's a repository](https://github.com/vtu-dog/tauri-bugtest) with sample code and videos showcasing the bugs.

In certain situations, a white stripe (approximately 1px wide) appears on the sides and / or at the bottom of a window, ignoring window boundaries. I had to [record](https://github.com/vtu-dog/tauri-bugtest/blob/master/window-side.mov) this bug using a camera, as using QuickTime somehow makes it not appear at all.

### Reproduction

1. Open [this link](https://demo.ovenplayer.com/demo_input.html), type `wss://demo.ovenplayer.com:3333/app/test-input-stream?direction=send&transport=tcp` in the `WebRTC input URL` and press `Start`. This step is necessary, as the application does not exhibit any strange behaviour when no videos are playing. You might want to disable your camera for that step.
2. Clone the linked repository and run `cargo tauri dev`. You should see the stream that you started in the browser.
3. Mouse over the title bar and wait for a few seconds. A white stripe should appear at the bottom of the window, just over the window boundary. It does not matter if the window is in focus or not. 

### Expected behavior

White stripes should not appear.

### Platform and versions

```shell
Operating System - Mac OS, version 12.3.0 X64

Node.js environment
  Node.js - 17.7.1
  @tauri-apps/cli - 1.0.0-rc.6
  @tauri-apps/api - 1.0.0-rc.2

Global packages
  npm - 8.5.2
  pnpm - 6.11.0
  yarn - 1.22.17

Rust environment
  rustup - 1.24.3
  rustc - 1.59.0
  cargo - 1.59.0
  toolchain - stable-aarch64-apple-darwin

App directory structure
/dist
/src-tauri
/.git

App
  tauri - 1.0.0-rc.4
  tauri-build - 1.0.0-rc.4
  tao - 0.6.4
  wry - 0.13.3
  build-type - bundle
  CSP - unset
  distDir - ../dist
  devPath - ../dist
```

### Additional context

I tried to create a reproducible example where stripes appear on both sides of the window, but it only seems to work when I tune in to my friend's private stream. It's very fickle.

**Comment by JonasKruckenberg** [comment]
As those are most likely unrelated bugs, can you please open a separate issue for the secondary one?

**Comment by vtu-dog** [comment]
Done. The other issue can now be found here: #3801

**Comment by lucasfernog** [comment]
Looks like this issue is fixed in the latest Tauri, you can try updating your project to see it. I believe it was fixed by #3915
